contrib/debian/copyright

```diff
@@ -72,7 +72,6 @@ Files: src/qt/res/icons/tx_mined.png
        src/qt/res/src/hd_enabled.svg
 Copyright: Jonas Schnelli
 License: Expat
-Comment:
```

Comment:
This looks like a syntax change on the file format and not a typo?

Reply:
There is no typo here, just a deleted line because there was no actual comment there.

Reply:
Phrased differently: Can debian still parse the file, when this field is missing? Why was it added in the first place?

Reply:
Oh - looking at the file you can see that comments aren't added every time. So I just assumed it's safe to delete this line.

Reply:
ok, fine